src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -228,6 +243,7 @@ TEST_F(ApplicationManagerImplTest, ProcessQueryApp_ExpectSuccess) {
       .WillByDefault(Return(sptr));
   ON_CALL(*mock_message_helper_, CreateNegativeResponse(_, _, _, _))
       .WillByDefault(Return(sptr));
+  EXPECT_CALL(*mock_rpc_service_, ManageHMICommand(_)).WillOnce(Return(false));
```

Comment:
@ValeriiMalkov return true probably. And check arg of ManageHMICommand

Reply:
@LuxoftAKutsan Fixed in commit cb0fa9a5bd7649dc7322d420554dc10fdaf4adff